packages/create-pantheon-decoupled-kit/src/templates/next-drupal/.env.example

```diff
@@ -1,14 +1,27 @@
 # Copy as .env.development.local to override envars for local development
+# Optional environment variables are commented out
+
+# The URL of your Drupal site
 BACKEND_URL=http://my-backend-site.pantheonsite.io
-FRONTEND_URL=http://my-frontend-site.pantheonsite.io
```

Comment:
FRONTEND_URL is probably not commonly used, but it is still something that we honor in next.config.js, so I wonder if we shouldn't remove this. 

I actually has to look it up in our docs to remember what it does...

---

If your site is translated and you would like the hreflang metadata set correctly, you may set FRONTEND_URL to the URL of your frontend site.

If the FRONTEND_URL is not set, it will default to the value of PANTHEON_ENVIRONMENT_URL

---

My assumption is that this would still be necessary for translated production sites that have a live domain - I'm assuming that the PANTHEON_ENVIRONMENT_URL is the platform URL, but that might be worth revisiting.

Reply:
It is actually set automatically in the next.config! We could still add it here to call it out, but I don't mind "hiding" this one.